dgraphtest/config.go

```diff
@@ -163,3 +164,8 @@ func (cc ClusterConfig) WithExposedPortOffset(offset uint64) ClusterConfig {
 	cc.portOffset = int(offset)
 	return cc
 }
+
+func (cc ClusterConfig) WithFeatureFlags(val string) ClusterConfig {
```

Comment:
For now its okay, but it would cause issues as we increase the number of feature flags. Can we do setters and getters for this? I am okay with it being a todo for now as we don't need it just yet

Reply:
oh, I see. This is part of dgraphtest and you are right, we should improve this. But as you said, we can do that whenever we have more flags. This is internal API. Let me see if I can do something now.